### PR TITLE
charts/kminion - allow setting priorityClassName for kminion pod

### PR DESCRIPTION
**Pull request by Vince-Chenal**
Hey,

I've been using [cloudworkz/kafka-minion-helm-chart](https://github.com/cloudworkz/kafka-minion-helm-chart) and I was able to set `priorityClassName` on pods, which is not the case in this Chart.

Tell me if you expect anything else in the PR 🙏

Also I would need a new version to be released as soon as this PR is merged, should I change the version in `Chart.yaml` myself?

### CODE REVIEW

File: charts/kminion/templates/daemonset.yaml
Comment (RafalKorepta) [inline charts/kminion/templates/daemonset.yaml:50]: NIT: I'm more used to 
```suggestion
      {{- wiith .Values.priorityClassName }}
```

**Review by RafalKorepta — APPROVED** [review]

File: charts/kminion/templates/daemonset.yaml
Comment (Vince-Chenal) [reply charts/kminion/templates/daemonset.yaml]: done :+1:

**Comment by Vince-Chenal** [comment]
I'm not sure to understand the `check-values` check. Yes I'm adding a new root-level value
Am I doing anything wrong?

**Comment by RafalKorepta** [comment]
> I'm not sure to understand the check-values check. Yes I'm adding a new root-level value
Am I doing anything wrong?

No, it's fine. The check was invented for us to be always aware that values are changing, so that we should carefully check if such change is backward compatible. In your case, all is good.

**Comment by Vince-Chenal** [comment]
It seems that I won't be able to update branch, wait for checks approval and then merge :sweat_smile:

**Pull request merged**
